src/Compilers/Core/Portable/WellKnownMember.cs

```diff
@@ -508,6 +508,8 @@ internal enum WellKnownMember
         System_Runtime_CompilerServices_NativeIntegerAttribute__ctor,
         System_Runtime_CompilerServices_NativeIntegerAttribute__ctorTransformFlags,
 
+        System_Runtime_CompilerServices_PreserveBaseOverridesAttribute__ctor,
```

Comment:
>System_Runtime_CompilerServices_PreserveBaseOverridesAttribute__ctor [](start = 8, length = 68)

Similarly to the type, I think this should be a special member.

Reply:
Recorded in [https://github.com/dotnet/roslyn/issues/44208](https://github.com/dotnet/roslyn/issues/44208)

---
In reply to: [442335918](https://github.com/dotnet/roslyn/pull/44025#discussion_r442335918) [](ancestors = 442335918)